docs/en/v0.4/user-guide/clients/vector.md

```diff
@@ -9,6 +9,29 @@
 GreptimeDB uses gRPC to communicate with Vector, so the default port for the Vector sink is `4001`.
 If you have changed the default gRPC port when starting GreptimeDB with [custom configurations](../operations/configuration.md#configuration-file), use your own port instead.
 
+## Example
```

Comment:
We already have an example in line 7 🤣

Reply:
Well... That's great. Looks like I can close this PR 🫣